galasa-ui/src/components/test-runs/test-run-details/3270Tab/TabFor3270.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+'use client';
+import React, { useState } from 'react';
+import TableOfScreenshots from '@/components/test-runs/test-run-details/3270Tab/TableOfScreenshots';
+import DisplayTerminalScreenshot from '@/components/test-runs/test-run-details/3270Tab/DisplayTerminalScreenshot';
+import styles from '@/styles/test-runs/test-run-details/tab3270.module.css';
+import { TreeNodeData } from '@/utils/functions/artifacts';
+import ErrorPage from '@/app/error/page';
+import { TerminalImage } from '@/utils/interfaces/common';
+import { useTheme } from '@/contexts/ThemeContext';
+
+export default function TabFor3270({
+  runId,
+  zos3270TerminalData,
+}: {
+  runId: string;
+  zos3270TerminalData: TreeNodeData[];
+}) {
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [imageData, setImageData] = useState<TerminalImage | undefined>(undefined);
+
+  const current = useTheme().theme;
+  let theme: 'light' | 'dark';
+  if (current === 'system') {
+    if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+      theme = 'dark';
+    } else {
+      theme = 'light';
+    }
+  } else {
+    theme = current;
+  }
```

Comment:
Do we need to calculate the theme here? I thought the theme would be applied via a global property so we wouldn't need to constantly add `lightTheme` and `darkTheme` CSS classes everywhere